e2e/help-button.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { test, expect, type Page } from '@playwright/test';
+
+const testHelpButton = async (page: Page, videoLink: boolean) => {
+  const helpButton = page.getByTestId('get-help-dropdown');
+  await expect(helpButton).toBeVisible();
+  await helpButton.click();
+  await expect(page.getByTestId('get-hint')).toBeVisible();
+  await expect(page.getByTestId('ask-for-help')).toBeVisible();
+  if (videoLink) {
+    await expect(page.getByTestId('watch-a-video')).toBeVisible();
+  } else {
+    await expect(page.getByTestId('watch-a-video')).toBeHidden();
+  }
+};
+
+test.describe('help-button tests for a page with three links( help, hint and video ) ', () => {
+  let page: Page;
+  // visit the page with the video link
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    await page.goto(
+      '/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements'
+    );
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  test('should render the button, menu and the three links when video is available', async () => {
+    await testHelpButton(page, true);
```

Comment:
```suggestion
    await page.getByTestId('get-help-dropdown').click();
    await expect(page.getByTestId('get-hint')).toBeVisible();
    await expect(page.getByTestId('ask-for-help')).toBeVisible();
    await expect(page.getByTestId('watch-a-video')).toBeVisible();
```
This is what I recommend doing instead of using the helper.